src/GVF/GVF.jl

```diff
@@ -20,6 +20,12 @@ abstract type AbstractGVF end
 
 function get(gvf::AbstractGVF, state_t, action_t, state_tp1, action_tp1, preds_tilde) end
 
+get(gvf::AbstractGVF, state_t, action_t, state_tp1, preds_tilde) =
```

Comment:
Could you remind me what `preds_tilde` is referring to?

Reply:
Predictions for s_tp1. For Compositional questions.

Reply:
Do you think time indexing the name would make this more clear? For example calling it instead preds_tm1 for predictions made at the previous timestep?

Reply:
Yah, that would definitely be more clear for me personally. What do you guys think?

Reply:
Yup. preds_tilde is just hold over from td nets for me. I'll make this change.